src/server/config/env/runTime.js

```diff
@@ -52,16 +52,26 @@ const runTime = [
   {
     name: 'HTTP_PORT',
     normalize: (input) => {
-      const parsed = parseInt(input, 10);
-      // make sure the parsed value is the same value as input
-      // input may be a string or a number, we don't want === in this case, just ==
-      if (Number.isNaN(parsed) || parsed != input) { // eslint-disable-line eqeqeq
-        throw new Error(`env var HTTP_PORT needs to be a valid integer, given "${input}"`);
-      } else {
-        return parsed;
+      if (input) {
+        const parsed = parseInt(input, 10);
+        // make sure the parsed value is the same value as input
+        // input may be a string or a number, we don't want === in this case, just ==
+        if (Number.isNaN(parsed) || parsed != input) { // eslint-disable-line eqeqeq
+          throw new Error(`env var HTTP_PORT needs to be a valid integer, given "${input}"`);
+        } else {
+          return parsed;
+        }
       }
+      return undefined;
```

Comment:
why is this being returned? perhaps normalize shouldn't be called when `input` is falsey (an empty string)?